src/groundStation/groundStation.py

```diff
@@ -399,7 +399,7 @@ def getOptions(self):
             help='Satellite parameter for automatic programs (e.g FTP)')
 
         self.parser.add_argument(
-            '-f',
+            '-sched',
```

Comment:
It's not typical for single dash options to be longer than 1 character. Maybe the short form should just be removed